jaeger-agent-mixin/jaeger.libsonnet

```diff
@@ -10,10 +10,16 @@
   jaeger_mixin::
     if $._config.jaeger_agent_host == null
     then {}
+    else if $._config.jaeger_agent_host == 'status.hostIP' then
+      container.withEnvMixin([
+        { "name": "JAEGER_AGENT_HOST", "valueFrom": {"fieldRef":{"fieldPath":"status.hostIP"}}},
+        container.envType.new('JAEGER_TAGS', 'namespace=%s,cluster=%s' % [$._config.namespace, $._config.cluster]),
+        container.envType.new('JAEGER_SAMPLER_MANAGER_HOST_PORT', 'http://$(JAEGER_AGENT_HOST):5778/sampling'),
```

Comment:
this will break remote sampling b/c it's not providing the correct ip from the k8s downward API